doc/guides/bluetooth/mesh-pics.rst

```diff
@@ -0,0 +1,312 @@
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_1_1	True		Core Speciciation 4.0 or later version (M)
+==============	==============	=======================================
+
+
+Roles
+=====
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_2_1	True		Node (C.1)
+TSPC_MESH_2_2	False (*)	Provisioner (C.1)
+==============	==============	=======================================
+
+
+Node Capabilites - Bearers
+==========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_3_1	True		Advertising Bearer (C.1)
+TSPC_MESH_3_2	True		GATT Bearer (C.1)
+==============	==============	=======================================
+
+
+Provisioning
+============
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_4_1	True		PB-ADV (C.1)
+TSPC_MESH_4_2	True		PB-GATT (C.2)
+TSPC_MESH_4_3	True		Device UUID (C.3)
+TSPC_MESH_4_4	True		Sending Unprovisioned Device Beacon (C.4)
+TSPC_MESH_4_5	True		Generic Provisioning Layer (C.3)
+TSPC_MESH_4_6	True		Provisioning Protocol (C.3)
+TSPC_MESH_4_7	False (*)	Provisioning: Public Key OOB (C.5)
+TSPC_MESH_4_8	True		Provisioning: Public Key No OOB (C.5)
+TSPC_MESH_4_9	True		Provisioning: Authentication Output OOB (C.6)
+TSPC_MESH_4_10	False (*)	Provisioning: Authentication Input OOB (C.6)
+TSPC_MESH_4_11	False (*)	Provisioning: Authentication Static OOB (C.6)
+TSPC_MESH_4_12	True		Provisioning: Authentication No OOB (C.3)
+==============	==============	=======================================
+
+
+Node Capabilites - Network Layer
+================================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_5_1	True		Transmitting and Receiving Secured Network
+				Layer Messages (M)
+TSPC_MESH_5_2	True		Relay Feature (C.1)
+TSPC_MESH_5_3	True		Network Message Cache (C.2)
+==============	==============	=======================================
+
+
+Node Capabilites - Lower Transport Layer
+========================================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_6_1	True		Transmitting and Receiving a Lower Transport
+				PDU (M)
+TSPC_MESH_6_2	True		Segmentation and Reassemply Behavior (M)
+TSPC_MESH_6_2	False (*)	Friend Cache (C.1)
```

Comment:
We do support acting as a Friend, and that includes the Friend Cache. So I think this should be enabled.